winui/Circular-Charts/Appearance.md

```diff
@@ -11,26 +11,9 @@ documentation: ug
 
 The appearance of [SfCircularChart](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.SfCircularChart.html) can be customized by using the predefined palettes, custom palettes and gradient, which allows to enrich the application.
 
-## Palettes
+## Default PaletteBrushes
 
-Circular series provides options for applying various kinds of palettes. Some predefined palettes, such as.
-
-* Metro
-* AutumnBrights
-* FloraHues
-* Pineapple
-* TomatoSpectrum
-* RedChrome
-* PurpleChrome
-* BlueChrome
-* GreenChrome
-* Elite
-* LightCandy
-* SandyBeach
-
-### Applying Palette
-
-Each palette applies a set of predefined brushes to the series in a predefined order. [Metro](https://help.syncfusion.com/cr/winui/Syncfusion.UI.Xaml.Charts.ChartColorPalette.html#Syncfusion_UI_Xaml_Charts_ChartColorPalette_Metro) palette is the default palette.
+Default palette applies a set of predefined brushes to the series in a predefined order.
 
```

Comment:
@naveenkumar-sanjeevirayan : Change the sentence.

Reply:
@naveenkumar-sanjeevirayan  : Refer below PR.
https://github.com/syncfusion-content/winui-docs/pull/274/files#diff-85d8087422682cdf5ec05e9a5388808041e311ae9dd4147a8b8a128299bfec87

Reply:
content changes updated.